api/v1_projects.inc

```diff
@@ -186,6 +186,7 @@ function get_project_fields_with_attr(?string $attr, ?Project $project): array
         "last_edit_time" => ["sql_name" => "t_last_edit"],
         "smoothread_deadline" => ["sql_name" => "smoothread_deadline"],
         "forum_url" => ["sql_name" => "forum_url"],
+        "round_type" => ["sql_name" => "state"],
```

Comment:
This seems a bit confusing as round type is not the same as state. Some other cases do not correspond exactly either e.g. post_processor. Should we consider revising this somehow?

Reply:
changed it thus.